utils/chaos_recommender/chaos_recommender.py

```diff
@@ -110,8 +149,15 @@ def main():
     logging.info("Starting Analysis ...")
     logging.info("Fetching the Telemetry data")
 
-    file_path = prometheus.fetch_utilization_from_prometheus(prometheus_endpoint, auth_token, namespace, scrape_duration)
-    analysis(file_path, chaos_tests)
+    file_path, queries = prometheus.fetch_utilization_from_prometheus(prometheus_endpoint, auth_token, namespace, scrape_duration)
+    analysis_data = analysis(file_path, chaos_tests, json_output)
+
+    if json_output is True:
+        time_str = time.strftime("%d%m%y_%H%M%S", time.localtime())
+        recommendation_json = f"recommender_{namespace}_{time_str}.json"
+        make_json_output(recommendation_json, namespace, kubeconfig, prometheus_endpoint, chaos_tests, scrape_duration, queries, analysis_data)
+        logging.info(f"Recommendation output is in file {recommendation_json} in chaos recommender output folder.")
```

Comment:
We might want to also print the json to stdout for user to refer. Thoughts?

Reply:
All the information is printed in the terminal when the recommender runs, so I am either not sure if it's unnecessary or if I just do not understand what you mean

Reply:
Maybe we should consider printing just the json vs text when json_output flag is set. Thoughts?